packages/catalog-model/examples/acme/org.yaml

```diff
@@ -30,3 +30,4 @@ spec:
     - ./team-b-group.yaml
     - ./team-c-group.yaml
     - ./team-d-group.yaml
+    - ./team-e-big-name-group.yaml
```

Comment:
i think it's unfortunate to add this entity here for no reason other than demonstrating that this was fixed - i vote to remove it

Reply:
removed